indico/web/client/js/react/components/DatePicker.jsx

```diff
@@ -96,11 +101,15 @@ DatePicker.propTypes = {
   format: PropTypes.string,
   onChange: PropTypes.func.isRequired,
   value: PropTypes.any,
+  min: PropTypes.string,
```

Comment:
Shouldn't the min/max also be taken into account in `handleDateChange` (to make the field invalid if you manually enter something outside the range)?

Reply:
The way I see it, `DatePicker`'s responsibility is to facilitate date entry, not perform input validation. It only restricts the input to a specific format, and controls the display of the calendar. It does not perform validation.

Reply:
I guess we can do it in the FinalDatePicker then since that's where other invalid input it turned into a validation error as well...

Reply:
Added